pinot-common/src/main/java/org/apache/pinot/common/function/scalar/StringFunctions.java

```diff
@@ -228,4 +267,86 @@ public static String chr(int codepoint) {
     char[] result = Character.toChars(codepoint);
     return new String(result);
   }
+
+  /**
+   * @see StandardCharsets#UTF_8#encode(String)
+   * @param input
+   * @return bytes
+   */
+  @ScalarFunction
+  public static byte[] to_utf8(String input) {
```

Comment:
Don't put underscore in function signature, or it won't match function name without underscore or with underscore at a different place, e.g. `toUtf8` or `to_utf_8`